android/src/main/java/com/reactnativecommunity/webview/RNCWebView.java

```diff
@@ -242,12 +247,31 @@ public boolean getMessagingEnabled() {
         return this.messagingEnabled;
     }
 
-    protected RNCWebViewBridge createRNCWebViewBridge(RNCWebView webView) {
-        if (bridge == null) {
-            bridge = new RNCWebViewBridge(webView);
-            addJavascriptInterface(bridge, JAVASCRIPT_INTERFACE);
+    protected WebViewCompat.WebMessageListener bridgeListener = null;
+
+    protected void createRNCWebViewBridge(RNCWebView webView) {
+        if (this.bridgeListener == null &&
+                WebViewFeature.isFeatureSupported(WebViewFeature.WEB_MESSAGE_LISTENER)) {
```

Comment:
should we also support fallback if the feature is not available? maybe we should keep original code as a fallback path.

Reply:
Added the previous code path as fallback!